content/en/blog/news/2022-gsoc-followup.md

```diff
@@ -0,0 +1,84 @@
+We want to be transparent about the entire process.
+
+### How many contributors are we looking for?
+
+We have 3 mentors signed up at the moment and are looking to select maximum 2 applicants.
+
+We want to ensure we have enough time available to mentor individual contributors.
+We want you to have a great experience contributing to Jenkins X and learn lots of new things!
+
+### What are we looking for in an ideal Jenkins X contributor?
+
+- Willingness to learn Jenkins X and engage with the community
+- Independence and a drive to investigate and propose solutions to complex issues that might occur during implementation
+- (Optional) Eventually become a Jenkins X maintainer (and some day mentor new GSoC contributors)!
+
+### What can prospective contributors do? (March 7 - April 19)
+
+You do not have to complete all of these tasks.
+This is presented here so that you have some clarity around where to start and which resources to follow.
+
+- Recommended: Join the jenkins X community [slack channels](https://jenkins-x.io/community/#slack) and say hello :)
+  - We prefer you post messages in the jenkins-x-dev channel
+- Recommended: Interact with potential mentors to understand/review the requirements of the projects.
+- Recommended: Start working on your proposal early on.
+- Optional: Learn about [docker](https://youtu.be/3c-iBn73dDE)
+- Optional: Learn about [kubernetes](https://youtu.be/X48VuDVv0do)
+- Optional: Learn the basics of [golang](https://go.dev/tour/list)
+- Optional: Understand Jenkins X concepts and fundamentals by going over the [documentation](https://jenkins-x.io/)
+- Optional: Set up Jenkins X locally and give it a go (Please don't spend money running Jenkins X in the cloud).
+  - We prefer you use [k3s](https://jenkins-x.io/v3/admin/platforms/k3s/) for this, as it's less resource intensive and can be installed locally on your workstation (linux/mac).
+
+### Interview process and selection criteria (April 19 and May 12):
+
+We will start interviewing candidates between April 19 and May 12.
+We want to individually talk to you, and get to know you better.
+
+Contributors will be scored based on the following criteria:
+
+Important:
+
+- Independent thinker and willingness to learn
+- Project proposal demonstrating good understanding of the project they are interested to work on
+
+Nice to have:
+
+- Technical expertise (basic knowledge goes a long way):
+
+  - Past experience with container technology
+  - Past experience with kubernetes
+  - Past experience with golang
+  - Past experience with a Frontend UI library (if interested to contribute to the UI project)
+
+- Past open source contribution experience
+- Any level of familiarity with Jenkins X (or other CI/CD tools)
+- Complete one of the three tasks listed below:
+  - If you have already contributed to Jenkins X then bring forward the github links to the pull requests so they can be reviewed by your mentor, explain what you did and why you did it.
```

Comment:
If you have already contributed to Jenkins X, then share the GitHub links to your pull requests. Your mentor will be happy to discuss them with you and will want to know more about what you did and why you did it.